cmd/k3a/acr.go

```diff
@@ -0,0 +1,239 @@
+		secretName, _ := cmd.Flags().GetString("secret-name")
+		namespace, _ := cmd.Flags().GetString("namespace")
+
+		done := spinner.Spinner(fmt.Sprintf("Starting ACR '%s' creation in region '%s'...", acrName, region))
+
+		if err := acr.Create(acr.CreateArgs{
+			SubscriptionID: subscriptionID,
+			Cluster:        clusterName,
+			ACRName:        acrName,
+			Location:       region,
+			SKU:            sku,
+		}); err != nil {
+			done()
+			return fmt.Errorf("failed to create ACR: %w", err)
+		}
+
+		done()
+		fmt.Printf("ACR '%s' created successfully in region '%s'\n", acrName, region)
+
+		if createSecret {
+			fmt.Println("Downloading kubeconfig and creating imagePullSecret...")
+
+			// Download kubeconfig
+			kubeDone := spinner.Spinner("Downloading kubeconfig from Key Vault...")
+			if err := kubeconfig.Download(kubeconfig.DownloadArgs{
+				Cluster: clusterName,
+			}); err != nil {
+				kubeDone()
+				return fmt.Errorf("failed to download kubeconfig: %w", err)
+			}
+			kubeDone()
+			fmt.Println("Kubeconfig downloaded successfully")
+
+			// Create imagePullSecret
+			secretDone := spinner.Spinner(fmt.Sprintf("Creating imagePullSecret '%s'...", secretName))
+			if err := acr.CreateImagePullSecret(acr.ImagePullSecretArgs{
+				SubscriptionID: subscriptionID,
+				Cluster:        clusterName,
+				ACRName:        acrName,
+				SecretName:     secretName,
+				Namespace:      namespace,
+			}); err != nil {
+				secretDone()
+				return fmt.Errorf("failed to create imagePullSecret: %w", err)
+			}
+			secretDone()
+			fmt.Printf("ImagePullSecret '%s' created successfully in namespace '%s'\n", secretName, namespace)
+
+			fmt.Printf("\nYou can now use the ACR in your pods by referencing the imagePullSecret:\n")
+			fmt.Printf("  imagePullSecrets:\n")
+			fmt.Printf("  - name: %s\n", secretName)
+		}
+
+		return nil
+	},
+}
+
+var listACRCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List ACR instances for a cluster",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		subscriptionID, _ := cmd.Root().Flags().GetString("subscription")
+		if subscriptionID == "" {
+			return fmt.Errorf("--subscription flag is required (or set K3A_SUBSCRIPTION)")
+		}
+		clusterName, _ := cmd.Flags().GetString("cluster")
+		if clusterName == "" {
+			return fmt.Errorf("--cluster flag is required (or set K3A_CLUSTER)")
+		}
+
+		return acr.List(acr.ListArgs{
+			SubscriptionID: subscriptionID,
+			Cluster:        clusterName,
+		})
+	},
+}
+
+var createSecretCmd = &cobra.Command{
+	Use:   "create-secret",
+	Short: "Create imagePullSecret for an existing ACR",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		subscriptionID, _ := cmd.Root().Flags().GetString("subscription")
+		if subscriptionID == "" {
+			return fmt.Errorf("--subscription flag is required (or set K3A_SUBSCRIPTION)")
+		}
+		clusterName, _ := cmd.Flags().GetString("cluster")
+		if clusterName == "" {
+			return fmt.Errorf("--cluster flag is required (or set K3A_CLUSTER)")
```

Comment:
Are these actually using the common `K3A_*` env vars?  Doesn't look like they are wired up in these commands.